lighthouse-core/config/default-config.js

```diff
@@ -552,6 +554,7 @@ const defaultConfig = {
         {id: 'doctype', weight: 1, group: 'best-practices-browser-compat'},
         {id: 'charset', weight: 1, group: 'best-practices-browser-compat'},
         // General Group
+        {id: 'no-unload-listeners', weight: 1, group: 'best-practices-general'},
```

Comment:
feels like it could be a diagnostic too, any particular decision insights behind best practices we can leave for future git blame archeologists? :)

Reply:
> feels like it could be a diagnostic too, any particular decision insights behind best practices we can leave for future git blame archeologists? :)

I guess it kind of falls in with `uses-long-cache-ttl` as something diagnosing performance, just not the performance of any of the metrics just measured (since a Lighthouse run isn't (usually) with a warm cache or on a hit of the back button). I guess I could see the argument that it's still worthwhile to have there, but I chose this as the uncontroversial spot to put it. I don't feel strongly :)